ESPixelStick/src/platformDefinitions/GPIO_Defs_ESP32_DevkitC.hpp

```diff
@@ -32,17 +32,17 @@
 #define DEFAULT_SPI_DATA_GPIO   gpio_num_t::GPIO_NUM_16
 #define DEFAULT_SPI_CLOCK_GPIO  gpio_num_t::GPIO_NUM_17
 
-#define DEFAULT_I2C_SDA         gpio_num_t::GPIO_NUM_3
-#define DEFAULT_I2C_SCL         gpio_num_t::GPIO_NUM_5
+#define DEFAULT_I2C_SDA         gpio_num_t::GPIO_NUM_21
+#define DEFAULT_I2C_SCL         gpio_num_t::GPIO_NUM_22
 
-// File Manager
+// File Manager - FIXED JMT
```

Comment:
Please dont add such identifiers to the source code. Git blame provides this without cluttering up code.